Fork update

fabrice-v opened the pull request:
You can merge this PR to run v1.3.1 of the validator. Unfortunately, the file touched in this fork was deleted in our repository hence it can't be merge automatically. I guess in that case it's up to you how to proceed further. We would recommend not to commit directly on `master` even in your own fork to help us provide you the best possible support.
Thks,

Commit:
Issues and Pull Request templates rework (#461)

*Issues and PR templates rework

Co-authored-by: Cristhian-HA <cristhian@mobilitydata.org>

Commit:
Update docker.yml

Commit:
Update end_to_end.yml

Commit:
Final documentaiton update for v1.3.1 -- Deep Dive

unreasonableman commented:
@fabrice-v I forked less than two hours ago. bug_report.md was deleted since then? Why was it deleted?

ghost commented:
That would be earlier today yes in preparation for our deep dive presentation
https://github.com/MobilityData/gtfs-validator/commit/a9145a22f8e26fbbf86de5d0398e1bd5be75deb4 via https://github.com/MobilityData/gtfs-validator/pull/461

barbeau committed:
docs: Fix GitHub Actions section to reference the right step (#464)

Commit:
One last doc fix

Commit:
Still one more

Commit:
Fix typos

Commit:
Adding STM

barbeau committed:
docs: Fix typo in workflow comment (#463)

* docs: Fix typo in workflow comment in test_pack_doc.yml

Co-authored-by: fabrice-v <55884852+fabrice-v@users.noreply.github.com>

lionel-nj committed:
Gracefully handle NPE thrown in #397 (#469)

* implement new notice UnsupportedGtfsStructureNotice

* rework rule E054 to gracefully handle NPE by notice generation

* Add #397 dataset to end to end

* Broken link

@mcplanner, you could help not needing to clone me by providing direct links to .zip file in issues. They should open a .zip download dialog in a web browser.

Thank you for your collaboration,

* fixing url

Co-authored-by: fabrice-v <55884852+fabrice-v@users.noreply.github.com>

lionel-nj committed:
False positive on missing calendar_date field (#465)

* implement new notice MalformedCsvRowException
* implement custom exception MalformedCsvRowException
* additional unit tests

Co-authored-by: fabrice-v <55884852+fabrice-v@users.noreply.github.com>
Co-authored-by: Fabrice V <fabrice@mobilitydata.org>

Commit:
Remove STM

Out until #474 is closed

Commit:
URL from OpenMobilityData BART GTFS --> empty zip (#462)

* fix #462 - Modify use case rule to exclude files contained in __MACOSX directory

- modify rule logic
- draft additional unit tests

* Inject ZipFile in use case to ease testing

* Inject ZipFile in use case constructor to ease testing

- remove not used constructor parameter: execParamRepo

* Inject RawFileInfoBuilder in use case constructor to ease testing

* Fix #462 - Implement additional unit tests

- add test resources

* update documentation

* Update .github/workflows/end_to_end.yml


Co-authored-by: lionel-nj <lionel@mobilitydata.org>
